### DISCUSSION TITLE
Citation

### DISCUSSION THREAD

**Discussion by arklumpus** [AvaloniaUI/Avalonia#13076] [Q&A] [answered]
Thank you all for the great work, I'm now close to submitting a scientific paper describing my software created using Avalonia. But now I'm wondering, is there a preferred citation for Avalonia in scientific papers?

I suppose I could cite the website [https://avaloniaui.net/](https://avaloniaui.net/), but who should be named as the authors? I'm struggling between a few different options:
1. `AvaloniaUI OÜ` (as in the footer for the website)
2. `Avalonia Team` (as in the "Author" field for the NuGet package)
3. `kekekeks, wieslawsoltes, grokys, danwalmsley` (as in the "Owners" of the NuGet package)
4. `Tsukanov N, Šoltés W, Kirk S, Walmsley D` (same as above, but with real names rather than nicknames)
5. Something else?

Also, is there a more citable source (e.g. with a DOI or something similar)?

**Comment by MikeCodesDotNET** [comment, marked as answer]
Maybe 'AvaloniaUI organization'?

**Reply by arklumpus** [reply]
Great, thank you! I suppose I cannot get a more authoritative answer than from the CEO of Avalonia 😅

In case others find it useful, here is what my citation looks like:

> AvaloniaUI Organization. (2013). Avalonia UI. https://avaloniaui.net/
>
> AvaloniaUI Organization. Avalonia UI. 2013. https://avaloniaui.net/. Accessed 18 Aug 2023.

BibTeX entry:

```BibTex
@misc{,
   author = {AvaloniaUI Organization},
   title = {Avalonia UI},
   url = {https://avaloniaui.net/},
   year = {2013},
}
```

**Comment by thevortexcloud** [comment]
As with most open source software. It's fairly complicated to attribute to every individual author (mature OSS can have thousands of authors and many are not backed by any company). Personally I am in favour of "AvaloniaUI contributors" as it's fairly inclusive.
